resources/js/store/store.js

```diff
@@ -1,11 +1,58 @@
+import axios from "axios";
 import Vue from "vue";
 import Vuex from "vuex";
+import router from '../router/router.js';
 
 Vue.use(Vuex);
 
 export default new Vuex.Store({
-    state: {},
-    mutations: {},
-    getters: {},
-    actions: {}
+    state: {
+        user: JSON.parse(localStorage.getItem('user')),
```

Comment:
Nu kan je user dus undefined zijn, is het niet handiger om als default een {} mee te geven?

JSON.parse(localStorage.getItem('user')) || {}